ChronoAPI/ChronoLog/include/ChronoLogAdminRPCVisor.h

```diff
@@ -12,15 +12,19 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include "macro.h"
+#include "errcode.h"
 #include "RPCFactory.h"
 #include "ClientRegistryInfo.h"
 #include "ClientRegistryManager.h"
 
-class ChronoLogAdminRPCProxy {
+extern std::shared_ptr<ClientRegistryManager> g_clientRegistryManager;
```

Comment:
Do you want to combine AdminRPC proxy and MetadataRPC proxy into a single class and invoke RPC instance once? I think it is safe to assume that a client calls Chronolog metadata RPCs only after it connects to a server.

Reply:
I will push it for later.

Reply:
I did it anyway in https://github.com/scs-lab/ChronoLog/commit/e2e76404348bbf333452eede46734ff062c2d490.

Reply:
ok